core/resources/shaders/polyline.vs

```diff
@@ -69,7 +70,7 @@ void main() {
         vec4 extrude = UNPACK_EXTRUSION(a_extrude);
         float width = extrude.z;
         float dwdz = extrude.w;
-        float dz = u_map_position.z - abs(u_tile_origin.z);
+        float dz = u_map_position.z - u_tile_origin.w;
```

Comment:
shouldn't this be using style zoom?:
`float dz = u_map_position.z - u_tile_origin.z`?